### PR TITLE
making TransactionID part of logging, making TransactionId a uuid

### PR DESCRIPTION
**Pull request by mjburghoffer**

### CODE REVIEW

File: soa-logging/src/main/java/com/relateiq/soa/logging/TransactionIdModule.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+
+import java.io.IOException;
+import java.util.Random;
+
+public class TransactionIdModule extends AbstractModule {
+
+  @Override
+  protected void configure() {}
```
Comment (relateiq-ec2) [inline soa-logging/src/main/java/com/relateiq/soa/logging/TransactionIdModule.java]: MAJOR Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)

**Comment by relateiq-ec2** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by relateiq-ec2** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by jontg** [comment]
Nice work, and thanks for the quick turnaround!

**Pull request merged**